gcloud-java-dns/src/main/java/com/google/gcloud/dns/Dns.java

```diff
@@ -244,7 +218,7 @@ public static RecordSetListOption type(RecordSet.Type type) {
   /**
    * Class for specifying zone field options.
    */
-  class ZoneOption extends AbstractOption implements Serializable {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.](https://www.codacy.com/app/mziccard/gcloud-java/file/2331164897/issues/source?bid=3123103&fileBranchId=3204660#l221)